AsyncDisplayKit/ASMapNode.mm

```diff
@@ -190,6 +183,23 @@ - (void)resetSnapshotter
   _snapshotter = [[MKMapSnapshotter alloc] initWithOptions:_options];
 }
 
+- (void)createInitialOptions
+{
+  _options = [[MKMapSnapshotOptions alloc] init];
+  //Default world-scale view
+  _options.region = MKCoordinateRegionForMapRect(MKMapRectWorld);
+  _options.size = self.calculatedSize;
+}
+
+- (void)applySnapshotOptions
+{
+  [_mapView setCamera:_options.camera animated:YES];
```

Comment:
This should check if _options is nil, and either early return / not make any calls, or set default values.  Messaging nil for structure return values is actually not safe in Obj-C; it may be that your call points to this are gated, but we should do this anyway to protect against future calls being added.